packages/package_info_plus/lib/package_info_plus_tizen.dart

```diff
@@ -15,6 +15,7 @@ import 'package:package_info_plus_platform_interface/package_info_platform_inter
 /// being set to Linux's.
 /// https://github.com/fluttercommunity/plus_plugins/blob/c7c942d1d19595a536c6ae18168d3781ddef7785/packages/package_info_plus/lib/package_info_plus.dart#L43-L55
 class PackageInfoPlugin extends PackageInfoPlatform {
+  /// Register dart plugin to inject Tizen plugin.
```

Comment:
The meaning is not clear. A Dart plugin refers to a plugin written in pure Dart.

Reply:
How about "Inject Tizen's implementation of the plugin."?

Reply:
```dart
/// Tells [PackageInfo] to use [MethodChannelPackageInfo] as the default instance of [PackageInfoPlatform].
```
should be enough I think, or you can move some comments from the class description to this method. Also a method comment should be written with an implied subject of _This method_, so it should be like "Injects" but not "Inject".